### PR TITLE
jolt 0.0.24 (new formula)

### PR DESCRIPTION
**Pull request by david-w-millar**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/jolt.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
+class Jolt < Formula
+  desc "JSON to JSON transformation library and cli written in java"
+  homepage "https://bazaarvoice.github.io/jolt/"
+  url "http://search.maven.org/remotecontent?filepath=com/bazaarvoice/jolt/jolt-cli/0.0.24/jolt-cli-0.0.24.jar"
+  sha256 "5a4bf4afb8bd01154aa2bcf89bde87a1b2623ae8b6901893a4d9f1a291689bbf"
+
+  depends_on :java
```
Comment (MikeMcQuaid) [inline Formula/jolt.rb]: Is there a specific version you can add here? Thanks!

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

File: Formula/jolt.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
+class Jolt < Formula
+  desc "JSON to JSON transformation library and cli written in java"
+  homepage "https://bazaarvoice.github.io/jolt/"
+  url "http://search.maven.org/remotecontent?filepath=com/bazaarvoice/jolt/jolt-cli/0.0.24/jolt-cli-0.0.24.jar"
+  sha256 "5a4bf4afb8bd01154aa2bcf89bde87a1b2623ae8b6901893a4d9f1a291689bbf"
+
```
Comment (dunn) [inline Formula/jolt.rb]: Can add `bottle :unneeded` since we're just installing the JAR.

File: Formula/jolt.rb [hunk 3]
```diff
@@ -0,0 +1,77 @@
+class Jolt < Formula
+  desc "JSON to JSON transformation library and cli written in java"
```
Comment (dunn) [inline Formula/jolt.rb]: Tiny nit, but should it be "CLI" and "Java" (capitalized)?

File: Formula/jolt.rb [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+          "quality" : {
+            "Id" : "quality",
+            "Range" : 5,
+            "Value" : 3
+          }
+        }
+      }
+    EOS
+
+    system "#{bin}/jolt transform jolt_spec.json input.json > actual_output.json"
```
Comment (dunn) [inline Formula/jolt.rb]: You can avoid writing to disk by using our `shell_output` method; e.g.:
```ruby
actual = shell_output("#{bin}/jolt transform jolt_spec.json input.json")

Comment (dunn) [inline Formula/jolt.rb]: Oh, I see below that jolt expects to be passed files.  Then you can do 
```ruby
(testpath/"actual.json").write shell_output("#{bin}/jolt transform jolt_spec.json input.json")

**Comment by dunn** [comment]
Looks like you pulled in some other commits.  Could you try rebasing against the current master branch?

**Comment by david-w-millar** [comment]
Oh boy.  Sloppy git fu.  Should I close this and re-open?

**Comment by david-w-millar** [comment]
Sure @dunn .  Sorry for that!

**Comment by dunn** [comment]
You can just force-push to this branch to update the PR.

**Comment by woodruffw** [comment]
Any update here @david-w-millar?

**Comment by david-w-millar** [comment]
Hey @woodruffw - I'll wrap this up in the next day or so.

**Comment by david-w-millar** [comment]
This pull request can be close.  This tool is no longer useful for me.  Sorry to waste your time!

**Pull request closed without merging**